### PR TITLE
cli: Check length of key-value pair for attributes

### LINKED ISSUE #804: CLI Panics when malformed attribute is input
#### Summary

CLI Panics when a malformed attribute is used on an `update`.

#### Steps to Reproduce

1.  Run `ttn-lw-cli gateways set eui-58a0cbfffe8008a0 --attributes a`

#### What do you see now?

```
panic: runtime error: index out of range

goroutine 1 [running]:
go.thethings.network/lorawan-stack/cmd/ttn-lw-cli/commands.mergeAttributes(0x0, 0xc0001a1100, 0xc0004d4a00)
    /home/travis/gopath/src/go.thethings.network/lorawan-stack/cmd/ttn-lw-cli/commands/flags.go:74 +0x2f8
go.thethings.network/lorawan-stack/cmd/ttn-lw-cli/commands.glob..func56(0x2b02ec0, 0xc000142060, 0x1, 0x3, 0x0, 0x0)
    /home/travis/gopath/src/go.thethings.network/lorawan-stack/cmd/ttn-lw-cli/commands/gateways.go:263 +0x22f
github.com/spf13/cobra.(*Command).execute(0x2b02ec0, 0xc000297f80, 0x3, 0x3, 0x2b02ec0, 0xc000297f80)
    /home/travis/gopath/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:762 +0x465
github.com/spf13/cobra.(*Command).ExecuteC(0x2b07ac0, 0x0, 0x0, 0x1c57b6c)
    /home/travis/gopath/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:852 +0x2ec
github.com/spf13/cobra.(*Command).Execute(...)
    /home/travis/gopath/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:800
main.main()
    /home/travis/gopath/src/go.thethings.network/lorawan-stack/cmd/ttn-lw-cli/main.go:27 +0x32
```
#### What do you want to see instead?

An error since the input in invalid but don't panic (pun intended).

#### Environment

```
The Things Network Command-line Interface: ttn-lw-cli
Version:             3.0.3
Go version:          go1.12.5
OS/Arch:             darwin/amd64
```

#### How do you propose to implement this?

Need to look into how `cobra` handles this.

#### Can you do this yourself and submit a Pull Request?

Yup

### PR DESCRIPTION
**Pull request by KrishnaIyer**
#### Summary

Closes #804 

#### Changes

- Added an valid length check while parsing attributes.

#### Release Notes

- Fixed CLI panic for invalid attributes.

### CODE REVIEW

File: cmd/ttn-lw-cli/commands/flags.go [hunk 1]
```diff
@@ -71,10 +71,12 @@ func mergeAttributes(attributes map[string]string, flagSet *pflag.FlagSet) map[s
 	}
 	for _, kv := range kv {
 		kv := strings.SplitN(kv, "=", 2)
-		if kv[1] == "" {
-			delete(out, kv[0])
-		} else {
-			out[kv[0]] = kv[1]
+		if len(kv) == 2 {
```
Comment (johanstokking) [inline cmd/ttn-lw-cli/commands/flags.go]: Early exit?

Comment (KrishnaIyer) [reply cmd/ttn-lw-cli/commands/flags.go]: It needs to still range though valid values, so this will just continue to the next item.

Comment (johanstokking) [reply cmd/ttn-lw-cli/commands/flags.go]: I mean there's a keyword for continue; `continue`

Comment (KrishnaIyer) [reply cmd/ttn-lw-cli/commands/flags.go]: lol. But this does exactly that without using `continue`.

**Comment by KrishnaIyer** [comment]
Ok done.. :)

**Pull request merged**